apps/institution-service/src/controllers/aggregatorIntegrationController.ts

```diff
@@ -15,6 +15,30 @@ interface AggregatorIntegrationParams {
   isActive: boolean;
 }
 
+function getErrorCodeAndMessage(error: Error): [number, string] {
+  if (error instanceof ValidationError) {
+    if (error.name === "SequelizeUniqueConstraintError") {
+      return [
+        409,
+        "An AggregatorIntegration for that Institution/Aggregator already exists. Cannot duplicate",
```

Comment:
I'd remove the cannot duplicate. It makes it seem like this is a duplicate request